### ISSUE TITLE
Sonoff Pow - Get Power Consumption Data every 1 second via MQTT - Implementability

### ISSUE THREAD

**Issue by oblansundar** [xoseperez/espurna#848]
I am using Sonoff POW to get the power consumption data. I have flashed it with Sonoff-Tasmota Firmware (5.13.1 - https://github.com/arendst/Sonoff-Tasmota) previously  and I have a similiar issue opened with Sonoff-Tasmota - https://github.com/arendst/Sonoff-Tasmota/issues/2655. 

So far, I get the Power Consumption data under the topic tele/<MQTT_CLIENT_ID>/SENSOR automatically for minimum interval of 10 seconds. This interval is the lower limit as suggested in https://github.com/arendst/Sonoff-Tasmota/wiki/Commands. 

**But then, if i use the firmware suggested here - https://github.com/xoseperez/espurna., then would it be possibile to get the data every one second?**

I just have a requirement that I need the information published under the topic tele/<MQTT_CLIENT_ID>/SENSOR automatically for one second if we meet certain thresholds as suggested here - #688 (In this thread, the telemetry interval is 30 seconds - my requirement is one second). I can get the SENSOR information under stat/<MQTT_CLIENT_ID>/STATUS10 , but I need to publish this topic every second to Sonoff POW and I ran a Python script to do this, but the MQTT got flooded. So, I needed to find an approach to do something in firmware that publishes MQTT under SENSOR topic for every one second.

Basically, to be accurate, I would like to have the same behavior as #688 (Checking the thresholds) except the telemetry interval I want is one second.

I have the following questions:
Is the above feasible? If so, how could it be implemented? Is it implementable with https://github.com/xoseperez/espurna firmware?
